js/everything.js

```diff
@@ -8,6 +8,10 @@ var timeIntervals = {
     SECONDS: 60
 };
 
+// global playlist, this is populated with an ajax call
+var playList = [];
```

Comment:
I don't know if playlist is the right name since this is really a list of upcoming "suggested" videos